src/Surfnet/ServiceProviderDashboard/Domain/ValueObject/DpaType.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/**
+ * Copyright 2023 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\ServiceProviderDashboard\Domain\ValueObject;
+
+use Surfnet\ServiceProviderDashboard\Domain\Entity\PrivacyQuestions;
+use function in_array;
+
+/**
+ * Representation of the different DPA types Data Processing Agreement
+ * That we currently allow.
+ *
+ * Note that:
+ * - The possible DPA varieties match the Manage coin:privacy:dpa_type enum options
+ * - The `service_has_own_dpa` enum is chosen as the `coin:privacy:dpa_type` default value,
+ *   as prescribed in the coin json schema definition.
+ *
+ * Data type definition of the coin:privacy:dpa_type from Manage:
+ *  "coin:privacy:dpa_type": {
+ *    "type": "string",
+ *    "enum": [
+ *      "dpa_not_applicable",
+ *      "dpa_in_surf_agreement",
+ *      "dpa_model_surf",
+ *      "dpa_supplied_by_service",
+ *      "other"
+ *    ],
+ *    "default" : "service_has_own_dpa",
+ *    "info": "Determines what DPA this service has to offer"
+ *  }
+ */
+class DpaType
+{
+    public const DPA_TYPE_NOT_APPLICABLE = 'dpa_not_applicable';
```

Comment:
Do all those constant have to be public?

Reply:
Good point, only the `default` was used outside of the VO scope. 

Updated